python/triqs_cthyb/tail_fit.py

```diff
@@ -24,6 +24,90 @@
 
 from triqs.gf.gf_fnt import fit_hermitian_tail_on_window, replace_by_tail
 
+from triqs.operators import c, c_dag
+from triqs.atom_diag import trace_rho_op
+
+
+def high_frequency_moments(density_matrix, 
```

Comment:
should we call it maybe sigma_high_freq_moments to make clear that this computes moments of Sigma?